runtime/src/main/scala/akka/grpc/AkkaGrpcException.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc
+
+import com.google.protobuf.any.Any
+import com.google.rpc.{ Code, Status }
+import io.grpc.StatusRuntimeException
+import io.grpc.protobuf.StatusProto
+import collection.JavaConverters._
+
+object AkkaGrpcException {
+
+  def apply(code: Int, message: String, details: Option[scalapb.GeneratedMessage]): AkkaGrpcException = {
```

Comment:
I think having a convenience constructor that takes code, message and details is a great idea.

For `code`, I think having `com.google.rpc.Code` is slightly neater compared to `Int`

For `details`, it might be nice to accept a `Seq`.

Reply:
This are some good ideas, especially the last one ;)